kubernetes/deployment-production-canary.tmpl

```diff
@@ -5,7 +5,6 @@ metadata:
   labels:
     app: panoptes-production-canary-app
 spec:
-  replicas: 0
```

Comment:
Preferably we set this as `replicas: 1` instead of removing so it's clear 1 is running and easy to set it back to 0 when we don't need it.